### PR TITLE
Add Characteristic Cycling Speed Cadence

### PR DESCRIPTION
**Pull request by joeblau**
Add CharacteristicCyclingSpeedCadence to SIG

### CODE REVIEW

File: Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    
+    
+    public enum Flags: UInt8 {
+        case wheelPresent = 0
+        case crankPresent = 1
+    }
+    
+    private(set) public var flags: Flags
+        
+    public enum PresentData {
```
Comment (khoogheem) [inline Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift]: I would rather see this as a actual Struct that is: Revolutions + lastEvent Time
and then have one var for wheel and one for crank

File: Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    public static var uuidString: String { "2A5B" }
+    
+    
+    /// Name of the Characteristic
+    public var name: String { Self.name }
+    
+    /// Characteristic UUID String
+    public var uuidString: String { Self.uuidString }
+    
+
```
Comment (khoogheem) [inline Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift]: space

File: Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+@available(swift 3.1)
+@available(iOS 10.0, tvOS 10.0, watchOS 3.0, OSX 10.12, *)
+final public class CharacteristicCyclingSpeedCadence: Characteristic {
+    
+    /// Characteristic Name
+    public static var name: String { " Cycling Speed and Cadence" }
+    
+    /// Characteristic UUID
+    public static var uuidString: String { "2A5B" }
+
```
Comment (khoogheem) [inline Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift:24]: space

File: Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+//  
+//
+//  Created by Joe Blau on 9/10/20.
+//
+
+import Foundation
+import DataDecoder
+import FitnessUnits
+
+/// BLE Alert Level Characteristic
```
Comment (khoogheem) [inline Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift:12]: fix

File: Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift [hunk 5]
```diff
@@ -0,0 +1,146 @@
+//
+//  File.swift
```
Comment (khoogheem) [inline Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift:2]: fix

File: Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+//  
+//
+//  Created by Joe Blau on 9/10/20.
+//
+
```
Comment (khoogheem) [inline Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift:7]: should have the terms

**Review by khoogheem — CHANGES_REQUESTED** [review]
a few nits..

**Comment by joeblau** [comment]
I accidentally deleted my other fork so I submitted a new PR (#7) with a live fork.

**Pull request closed without merging**